net-analyzer/nuttcp/files/nuttcp.initd

```diff
@@ -0,0 +1,16 @@
+#!/sbin/openrc-run
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+command="/usr/bin/nuttcp"
+command_background="yes"
+command_args="${NUTTCP_OPTS}"
+pidfile="/run/${RC_SVCNAME}.pid"
+
+depend() {
+	need net
+}
+
+start_post() {
+	pgrep -n $(echo ${RC_SVCNAME} | sed 's/\..*//g') > ${pidfile}
```

Comment:
Don't use `echo | sed`, use pattern substitution instead (it works on POSIX shell). Also quote `${pidfile}`.

Reply:
Done.